### PR TITLE
Bug 1833486: Bump fluentd and dependencies to 1.11.1

### PR DESCRIPTION
**Pull request by jcantrill**
This PR bumps dependencies and  fluent to 1.11.1 to address inconsistencies

Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1833486

### CODE REVIEW

**Comment by periklis** [comment]
@jcantrill I noticed this issue regarding performance loss (athough CPU usage related) that might affect out loss too: https://github.com/fluent/fluentd/issues/2871 --> Seems to be related only for msgpack and we vendor this too. Fix was merged in **1.10**. Maybe worth a try?

**Comment by jcantrill** [comment]
@periklis Thanks for the ref which seems to have resolved our issues.

File: fluentd/rh-manifest.txt [hunk 1]
```diff
@@ -1,83 +1,65 @@
...
-excon 0.73.0 https://github.com/excon/excon
-faraday 0.17.3 https://lostisland.github.io/faraday
-ffi 1.12.2 https://github.com/ffi/ffi/wiki
+digest-crc 0.6.1 https://github.com/postmodern/digest-crc#readme
+dig_rb 1.0.1 https://github.com/jrochkind/dig_rb
 ffi-compiler 1.0.1 http://wiki.github.com/ffi/ffi
 fluent-config-regexp-type 1.0.0 https://github.com/okkez/fluent-config-regexp-type
-fluentd 1.9.2 https://www.fluentd.org/
-fluent-mixin-config-placeholders 0.4.0 https://github.com/tagomoris/fluent-mixin-config-placeholders
+fluentd 1.7.4 https://www.fluentd.org/
```
Comment (periklis) [inline fluentd/rh-manifest.txt:5]: Why does the script think we use 1.7.4 here?

Comment (jcantrill) [reply fluentd/rh-manifest.txt]: updated to truly be 1.11.1

**Pull request merged**